docs/quickstart.rst

```diff
@@ -1,5 +0,0 @@
-##########
-Quickstart
-##########
-
-See :ref:`quickstart`.
```

Comment:
Maybe this could be: See :ref:`getting_started`.

This page is here because there were pages that linked to quickstart from outside the docs here, and it went to a broken page. If we keep this and have a basic redirect, users that land here can find their way to the getting started guide.

Reply:
@nvkevlu got it, in that case, maybe we should update those documentation to point to getting started page?

Reply:
The problem is some of those were published and linked outside of our control, so on our end, if someone does arrive at the page, it is better to exist and have a link pointing to something rather than the page does not exist error.

Reply:
got it, let me add it back.